airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -168,7 +170,14 @@ public SourceConnection getSourceConnection(final UUID sourceId) throws JsonVali
     return persistence.getConfig(ConfigSchema.SOURCE_CONNECTION, sourceId.toString(), SourceConnection.class);
   }
 
-  public void writeSourceConnection(final SourceConnection source) throws JsonValidationException, IOException {
+  public void writeSourceConnection(final SourceConnection source, final ConnectorSpecification connectorSpecification)
+      throws JsonValidationException, IOException {
+    System.out.println("connectorSpecification = " + connectorSpecification);
```

Comment:
I'm going to clear this out before merging if this is approved. This is just to verify for now that an actual valid connector specification is being passed in so it'll be easier to handle future PRs.